Source/EbsdLib/EbsdReader.h

```diff
@@ -207,8 +207,11 @@ class EbsdLib_EXPORT EbsdReader
 
 
   private:
+  public:
```

Comment:
`private: public:` should probably just be `public:`
Consider deleting one of the empty lines above this.